src/wallet/wallet.h

```diff
@@ -723,6 +723,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      */
     mutable CCriticalSection cs_wallet;
 
+    std::atomic<bool> fAbortRescan;
```

Comment:
I would turn both these flags to private standard boolean and use CWallet getters&setters to change them